CRM/Extension/Browser.php

```diff
@@ -157,21 +157,31 @@ public function getExtensions() {
 
   /**
    * Get a description of a particular extension.
+   * This function is an optimised form of getExtensions() - it returns the
```

Comment:
I meant that by

> This function is an optimised form of getExtensions()

It sounds like it's another version of it. Whereas for someone using this class it should be pretty clear what the methods are. But it's not a big issue for me so leave it as it is if you like.